components/script/dom/htmltableelement.rs

```diff
@@ -51,6 +54,31 @@ impl HTMLTableElement {
 }
 
 impl HTMLTableElementMethods for HTMLTableElement {
+    // https://html.spec.whatwg.org/multipage/#dom-table-rows
+    fn Rows(&self) -> Root<HTMLCollection> {
+        #[allow(moved_no_move)]
```

Comment:
I'm not sure if there are any workarounds with this. If not, we'll have to properly refactor and implement how we want iterators to work in HTMLCollections.